packages/form_bloc/lib/src/blocs/field/field_bloc.dart

```diff
@@ -162,6 +164,7 @@ abstract class SingleFieldBloc<
     final _onFinish = onFinish ?? (State p, State c, R r) {};
 
     return stream
+        .startWith(_initialState)
```

Comment:
It should be more correct
```suggestion
        .startWith(state)
```